server/src/api/v2/organizations/get-organizations.ts

```diff
@@ -0,0 +1,69 @@
+import { Request, Response } from 'express';
+import { FindOptions } from 'sequelize';
+import { buildQuery, QueryBuilderConfig } from '../../../functions/query-builder.func';
+import { wrapResponse } from '../../../functions/response-wrapper';
+import { News } from '../../../models/news.model';
+import { Vars } from '../../../vars';
+import { OrganizationData } from '../../../interfaces/organization.interface';
+import { User } from '../../../models/user.model';
+import { Organization } from '../../../models/organization.model';
+import { Team } from '../../../models/team.model';
+import { Role } from '../../../models/role.model';
+import { Poll } from '../../../models/poll.model';
+
+
+
+
+export async function getOrganization(req: Request, res: Response): Promise<Response> {
+    let success = true;
+
+
+    const organizationData: OrganizationData | null = await Organization
+    .scope([Vars.currentUser.is_admin ? 'full' : 'active']) //  todo permission check
```

Comment:
todo erledigt?
Falls nein, was muss man hier noch genau machen? Bitte für später dann genauer beschreiben

Reply:
@TomasKostadinov kannst du das genauer beschreiben? Das war ja das was wir gemeinsam/du so geändert hast.

Reply:
Es muss noch geklärt werden, inwieweit a) Daten einer Organisation gesehen werden dürfen und b) ob für Aussenstehende überhaupt das sichtbar ist, dass es eine gibt

Reply:
Ich würde sagen: Klären wir später in #98